packages/reporters/dev-server/src/HMRServer.js

```diff
@@ -272,9 +272,19 @@ export default class HMRServer {
   }
 
   broadcast(msg: HMRMessage) {
-    const json = JSON.stringify(msg);
-    for (let ws of this.wss.clients) {
-      ws.send(json);
+    if (msg.type === 'update' && msg.assets.length > 10000) {
```

Comment:
Is this just an arbitrary threshold that should be well under the limit?

Also, should this be extracted to a `const` rather than a magic number as it's referenced in a few places?

Reply:
yeah, it's mostly arbitrary, just the most round number, which was around 75% of the limit when I was local testing.

Happy to move this to a constant 👍